### ISSUE TITLE
Accessibility Issue with PeoplePicker component

### ISSUE THREAD

**Issue by TianyuHou** [microsoft/fluentui#14477]
PWD Impact:

Visually challenged users who rely on screen readers are unable to use People picker, as NVDA is not navigating to the People picker field when navigated using form mode(f) or browse mode(down arrow key).

 

Test Environment:

OS : Win 10 2004 (OS Build 19041.329)

Browser:  New Edge 83.0.478.58

URL: https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker

Tools used: NVDA 2020.1

 

Repro Steps:

Step 1: Enable screen reader NVDA and open the above URL in New Edge browser.

Step 2: People picker screen will be displayed.

Step 3: Now navigate through the page in form mode(f) or browse mode(down arrow ) and verify whether NVDA is navigating to people picker field.

 

Actual Result:

NVDA is not navigating to the people picker field when navigated using form mode(f) or browse mode(down arrow key). NVDA is navigating only to the label and not moving to the edit field. (Only the ListPeoplePicker component can get navigated and open the combobox when using NVDA form mode.)

 

Expected Result:

NVDA should navigate to the people picker field when navigated using form mode(f) or browse mode(down arrow key).

 
MAS References:

MAS 1.3.2 - Meaningful Sequence

WCAG Reference:

https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

**Comment by paulgildea** [comment]
@smhigley FYI

**Comment by smhigley** [comment]
This is one of multiple support issues with the ARIA 1.1 combobox pattern. This particular one was fixed in NVDA 2020 with Firefox, but not with Chromium browsers.

The best solution would be to move to the ARIA 1.2 combobox pattern, which would involve putting `role="combobox"` and all related ARIA attributes directly on the `<input>` element instead of having a separate combobox parent.

**Comment by dikshantbansal98** [comment]
We need to launch a website in some days. This accessibility related issue is creating problems. Will this issue be fixed earlier or will take a long time?

**Comment by JustSlone** [comment]
@dikshantbansal98, yes sorry it will take some time, this is a tricky issue.
